OfficeApp3Web/Content/AppV2.css

```diff
@@ -1,541 +1,575 @@
+    padding-right: 30px;
+    padding-left: 20px;
+    bottom: 45px;
+}
+
+#codeMenu a.selected {
+     color: #337147;
+     font-weight: 600;
+
+}
+
+#samplesContainer {
+     overflow: hidden; 
+     display:flex;
+     flex-direction: column; 
+     position: relative;     
+     top:0px;
+     bottom: 0px; 
+     left: 0px; 
+     right: 0px; 
+     padding: 10px;
+}
+
+#samplesContainer select{
+    width:auto;  
+    -moz-appearance:none;
+    -webkit-appearance:none;
+    background:url("/Images/arrow.png");
+    background-repeat: no-repeat;
+    background-position: right; 
+    padding-right: 14px;
+}
+
+#samplesContainer select::-ms-expand{
+    display:none;
+}
+
+
+#goButton {
+    border:none;
+    cursor:pointer;
+    background:none;
+    font-size:large; 
+}
+
+#console {
+    width:100%; 
+    height:100px; 
+    bottom:0px;   
+}
+
+.divWrapperOutside {
+    border:none;
+    background-color: white;
+    position:relative;
+    left: 0px;
+    right: 0px;
+    top: 0px;  
+}
+
+.divWrapperInside {
+    position: relative;
+    background-color: white;     
+    height: 60%;    
+}
+
+.divWrapperInsideFull {
+    position: relative;
+    background-color: white;     
+    height: 100%;    
+}
+
+#buttonWrapper {
+   position:absolute;
+   bottom:0px;
+   height:50px;
+   width:100%;
+   background-color:white;
+}
+
+.snippetbuttonclass {   
+    background-color:#337147;
+    color: white;
+    padding: 5px 10px 5px 10px;
+    font-family:Segoe UI, Tahoma, Verdana, sans-serif;
+    border:solid;
+    border-color:#9E9E9E;
+    border-width:1px;
+    cursor:pointer;
+    margin:10px;
+}
+
+.codeSnippetsMenu {
+    background:none;
+    cursor:pointer;
+    border:none;
+    font-size:15px;
+    color:white;    
+    height: auto;
+    padding:0;
+}
+
+#scenariolabel {
+    width:auto;  
+    -moz-appearance:none;
+    -webkit-appearance:none;
+    background:url("/Images/arrowWhite.png");
+    background-repeat: no-repeat;
+    background-position: right; 
+    padding-right: 14px;
+}
+
+#scenariolabel::-ms-expand{
+    display:none;
+}
+
+.codeSnippetsMenu option{
+    color:black;
+}
+.codeSnippetsMenu .selected{
+    color:white;
+}
+
+#codeSnippet {
+    margin:10px;
+    margin-left:-50px;
+    width:180%;
+}
+.mainMenu {
+    height: auto;
+    background-color: #337147;
+    color: white; 
+}
+
+#sampleDescription {
+    height:auto;
+    padding:10px;
+}
+#sampleDescription p{
+    margin:5px;
+}
+
+#TxtRichApiScript {
+    height:70%; 
+    overflow: hidden; 
+    display:flex; 
+    flex-direction: column; 
+    position: relative; 
+    top:0px; 
+    bottom:0px; 
+    left: 0px; 
+    right: 0px; 
+    padding: 5px;
+}
+
+@media screen and (-ms-high-contrast:none){
+    #imgNext2 {
+        display:none;
+    }
+    #imgBack2{
+        display:none;
+    }
+}
+@media screen and (-ms-high-contrast:active)
+{
+     #sampleSelector{
+     overflow: hidden; 
+     background:url("/Images/arrow.png");

```

Comment:
I thought high contrast active won't pick up background image. Can you show me how it looks like?